src-plugins/medMaskApplication/medMaskApplication.cpp

```diff
@@ -156,7 +158,6 @@ void medMaskApplication::clearInput (int channel)
 
     if ( channel == 1 )
     {
-        d->output = NULL;
         d->input = NULL;
```

Comment:
If we clear the inputs, it means that we don't want the linked output also i think.